supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigAsyncJobExecutionActor.scala

```diff
@@ -304,7 +306,11 @@ class DispatchedConfigAsyncJobExecutionActor(override val standardParams: Standa
           jobLogger.error(s"Return file not found after ${exitCodeTimeout.getOrElse(backupError)} seconds, assuming external kill")
 
           val returnCodeTemp = jobPaths.returnCode.plusExt("kill")
-          returnCodeTemp.write(s"$SIGTERM${System.lineSeparator}")
+          // If SIGTERM, SIGKILL or SIGINT codes are used, cromwell will assume the job has been aborted by cromwell.
+          // And it will therefore NOT retry the job. Which makes perfect sense. Best not to change that in the
+          // StandardAsyncExecutionActor code, but give a nonsensical return code here.
+          val exitValue = if (retryExternallyKilledJobs) Int.MaxValue else SIGKILL
```

Comment:
This check looks redundant - you'll only get here if the `exitCodeTimeout` is defined, and that's the exact same condition as `retryExternallyKilledJobs `.

Reply:
:+1: I was tempted to put this in, in case we wanted to make it configurable if jobs should fail when externally killed. But expanding the code base for 'just in case' reasons is bad practice. Will remove this.